lib/rock_rms/resources/person.rb

```diff
@@ -3,22 +3,32 @@ class Client
     module Person
       def list_people(options = {})
         res = get(people_path, options)
-        RockRMS::Person.format(res)
+        RockRMS::Responses::Person.format(res)
       end
 
       def find_person(id)
         res = get(people_path(id))
-        RockRMS::Person.format(res)
+        RockRMS::Responses::Person.format(res)
       end
 
       def find_person_by_email(email)
         res = get("People/GetByEmail/#{email}")
-        RockRMS::Person.format(res)
+        RockRMS::Responses::Person.format(res)
       end
 
-      def find_person_by_name(full_name)
-        res = get("People/Search?name=#{full_name}&includeHtml=false&includeDetails=true&includeBusinesses=false&includeDeceased=false")
-        RockRMS::Person.format(res)
+      NAME_SEARCH_DEFAULTS = {
+        includeHtml: false,
+        includeDetails: true,
+        includeBusinesses: false,
+        includeDeceased: false
+      }.freeze
+
+      def find_person_by_name(full_name, options = {})
+        priority = options.merge(name: full_name)
+
+        RockRMS::Responses::Person.format(
+          get('People/Search', NAME_SEARCH_DEFAULTS.merge(priority))
```

Comment:
Is this a breaking change?

Reply:
This is not a breaking change because the options hash is optional. With no options passed, the request is exactly as it was previously. This way just takes advantage of the way `Faraday` lets you pass options to requests and also give flexibility to the lib user if they want to be more granular with their includes (whether Rock respects them is another story :trollface:)

Reply:
Lol

Reply:
I wrote specs for different scenarios, asserting the default behavior and for passing in specific options. https://github.com/taylorbrooks/rock_rms/pull/3/files#diff-1c2c3460b6776284cb543b705d4eb5eeR20

Reply:
v1.1 was just pushed up to Rubygems. 

Thanks for contributing!